src/components/SeqVizParamsModal.vue

```diff
@@ -322,16 +322,39 @@
                     if (this.params[name].visibleDependency) {
                         const dependsOn =
                             this.params[this.params[name].visibleDependency]
-                        if (
-                            dependsOn.value !== this.params[name].visibleValue
-                        ) {
-                            continue
+                        if ('visiblePredicate' in this.params[name]) {
```

Comment:
OK, let me know when you are done with the other commit(s) you are planning based on your review and I will work on a commit that will add an explanatory comment for each of the computed properties, and try to simplify the code in visibleParams so that it will be clearer and linebreak better.

Reply:
@gwhitney added some comments in https://github.com/numberscope/frontscope/pull/230/commits/e4b27f474aee11070f3dbb3ef83677915682a385.